src/pymatgen/io/vasp/outputs.py

```diff
@@ -2605,7 +2703,7 @@ def read_nmr_efg_tensor(self) -> list[NDArray]:
         """Parses the NMR Electric Field Gradient Raw Tensors.
 
         Returns:
-            A list of Electric Field Gradient Tensors in the order of Atoms from OUTCAR.
+            list[NDArray]: Electric Field Gradient Tensors in the order of atoms.
```

Comment:
double check type

Reply:
Cannot firm its type as I never do this calculation and no test is available, but should be correct as per the function implementation of `make_symmetric_matrix_from_upper_tri`

Reply:
I was wrong, test is available as `test_nmr_efg`